openfisca_us/variables/usda/snap/income/deductions/shelter_expense.py

```diff
@@ -0,0 +1,81 @@
+from openfisca_us.model_api import *
+
+
+class snap_shelter_deduction(Variable):
+    value_type = float
+    entity = SPMUnit
+    definition_period = YEAR
+    documentation = (
+        "Excess shelter deduction for calculating SNAP benefit amount"
+    )
+    label = "SNAP shelter deduction"
+    reference = ("United States Code, Title 7, Section 2014(d)(6)",)
```

Comment:
Is this right? I'm seeing that it points to child support: https://www.law.cornell.edu/uscode/text/7/2014#d_6

Also pointed to elsewhere in this PR.

Reply:
Good catch, it was (e) rather than (d)